cloud/deployment/inspect/inspect_test.go

```diff
@@ -825,7 +825,7 @@ func TestFormatPrintableDeployment(t *testing.T) {
         scheduler_count: 3
         workspace_name: test-ws
         deployment_type: STANDARD
-        cloud_provider: azure
+        cloud_provider: AZURE
```

Comment:
Is this a change worth documenting?

Reply:
I guess that is a breaking change if users are using the output of an old template to generate a deployment? 🤔

Reply:
CC: @sunkickr

Reply:
We should validate that "azure" still works as an input but yeah the output changing doesn't matter